main/src/test/java/org/mobilitydata/gtfsvalidator/validator/FeedServiceDateValidatorTest.java

```diff
@@ -63,31 +62,6 @@ public void startDateBeforeEndDateShouldNotGenerateNotice() {
     assertThat(noticeContainer.getValidationNotices().isEmpty());
   }
 
-  @Test
-  public void startDateAfterEndDateShouldGenerateNotice() {
```

Comment:
`startDateBeforeEndDateShouldNotGenerateNotice` the test just above this one should be removed too.

Reply:
Good catch!